crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -147,6 +140,18 @@ impl CodeGenerateable for EsmExports {
                         );
                     }
                 }
+
+                if let EcmascriptExports::CommonJs = &*asset.get_exports().await? {
+                    let ident = ReferencedAsset::get_ident_from_chunk(asset).await?;
+
+                    cjs_exports.push(PropOrSpread::Spread(SpreadElement {
+                        dot3_token: DUMMY_SP,
+                        expr: Box::new(quote!(
+                            "($cjs)" as Expr,
+                            cjs: Expr = Ident::new(ident.into(), DUMMY_SP).into()
+                        )),
+                    }));
```

Comment:
Spreading is not safe, as it makes a flat clone of the exports object.

But commonjs modules can change there properties at later time e. g.

```
exports.a = 1;

setTimeout(() => {
  exports.a = 2;
}, 1000);
```

You can call `__turbopack__cjs__` multiple times with all the exports objects.